### PR TITLE
<Input/> - On text click - select all

### PR DESCRIPTION
**Pull request by Faradey27**
### What changed

- added select on click on inpit

### Why it changed

- because of wsr-issue and to improve user experience 

---

- [x] Change is tested
- [x] Change is documented
- [ ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by Faradey27** [comment]
@hepiyellow will create e2e test

**Comment by hepiyellow** [comment]
@shlomitc This feature is missing in `wix-ui-core`, can we ask @Faradey27  to implement it there as well?
If so, then in the core, I would put this behaviour under a prop , `selectOnClickEnabled`.
We might want to have it as a prop also in WSR.

File: src/Input/Input.js [hunk 1]
```diff
@@ -192,6 +192,9 @@ class Input extends Component {
   };
 
   _onClick = e => {
+    if (e.target.value && e.target.select) {
```
Comment (hepiyellow) [inline src/Input/Input.js]: Should this select all only when the click is also focusing the component?

Comment (shlomitc) [reply src/Input/Input.js]: makes sense that also on focus

**Comment by hepiyellow** [comment]
Only on focus.

**Comment by Faradey27** [comment]
@hepiyellow @shlomitc i m closing pull request, because we already have such feature(auto select on focus) https://github.com/Faradey27/wix-style-react/blob/aa5e9fe4ce92f995f919997a8873e5a5972e225d/src/Input/Input.js#L183

**Comment by shlomitc** [comment]
We still need to make this behavior default

**Comment by Faradey27** [comment]
@shlomitc @hepiyellow updated with default props

**Pull request merged**